Tests/MongoSwiftTests/DNSSeedlistTests.swift

```diff
@@ -65,17 +65,29 @@ final class DNSSeedlistTests: MongoSwiftTestCase {
 
             // TODO: DRIVERS-796 or DRIVERS-990: unskip this test
             guard fileName != "txt-record-with-overridden-uri-option.json" else {
-                return
+                continue
```

Comment:
this was a bug too 😬 fortunately we pass all of the ones that came after with no issues.